src/Helper/Placeholder/Container/AbstractContainer.php

```diff
@@ -117,14 +117,17 @@ public function toString($indent = null)
             ? $this->getWhitespace($indent)
             : $this->getIndent();
 
-        $items  = $this->getArrayCopy();
         // If we don't have items - do not show prefix and postfix
-        if (!count($items)) {
+        if (!$this->count()) {
             return '';
         }
+
+        $itemsToString = implode($this->getSeparator(), $this->getArrayCopy());
+        // todo check empty $itemsToString after trim() to return ''
```

Comment:
If items with whitespace are added, then there is a reason for it and we should allow the rendering. Please undo this changes.